src/server/middleware/reactApplication/index.js

```diff
@@ -55,10 +56,17 @@ function reactApplicationMiddleware(request: $Request, response: $Response) {
     </CodeSplitProvider>,
   );
 
+  // hydrate glamor styles
+  let { css, ids } = renderStatic(() => app);
+
   // Generate the html response.
   const html = generateHTML({
     // Provide the full app react element.
     app,
+    // server-side generated glamor styles
+    css,
```

Comment:
Perhaps we should consider merging the css and id items into a "glamor" data object?